ruby_review_week_four.rb

```diff
@@ -1,37 +1,145 @@
 # You may NOT use Google. These questions are just to test where you are at. Get through as many of these as you can. If you don’t get through all of them it’s ok - they are designed to take longer than the available time. If you can’t remember a bit of code, write the pseudo-code that represents what you would write your Ruby code.
 
 # 1. What is a variable?
+#A variable is something that saves a data type 
+
 
 # 2. What Ruby method can you use to make a loop from an array?
+#.each
+
 
 # 3. In Ruby, what are booleans?
+#true or false
+
 
 # 4. How do classes relate to objects?
+#A class can create an instance object (which is an instance of that class)
+
 
 # 5. What is an instance variable?
+#An instance variable is a variable that can be shared (accessed) by all the methods of the class
+
 
 # 6. Choose an everyday item (cup, computer, person) and give it two instance variables (which will be attributes), and one method (to describe a function of the object). Code the initialize method in full (for these attributes only), and the definition line of the instance method (but do not code the entire method).
+class Person
+  def initialize(name, age)
+    @name = name
+    @age = age
+  end
+
+  def information
```

Comment:
this is great but make sure you follow the instructions precisely, no need to code the entire instance method